pkg/cmd/deletecmd/delete_application.go

```diff
@@ -481,16 +482,18 @@ func (o *DeleteApplicationOptions) init() error {
 	return nil
 }
 
-func (o *DeleteApplicationOptions) deletePipelineActivitiesForSourceRepository(jxClient versioned.Interface, ns string, srName string) error {
+func (o *DeleteApplicationOptions) deletePipelineActivitiesForSourceRepository(jxClient versioned.Interface, ns string, gitOwner string, gitRepo string) error {
+	gitOwnerSelector := fields.OneTermEqualSelector("metadata.spec.gitOwner", gitOwner)
```

Comment:
@abayer see whether this is going to cut it. If this gets us past the error with labels in the BDD tests, I can create a dedicated PR for #5515